test/e2e/installplan_e2e_test.go

```diff
@@ -418,6 +444,10 @@ var _ = Describe("Install Plan", func() {
 				Succeed(),
 				WithTransform(k8serrors.IsNotFound, BeTrue()),
 			))
+			Expect(ctx.Ctx().Client().Delete(context.Background(), plan)).To(Or(
```

Comment:
nit: wrap this with client.IgnoreNotFound(...) to avoid having to enforce this through gomega expressions?

Reply:
Fixed

Reply:
I'm still seeing it's using the gomega expressions?